Wikipedia/Code/WMFArticleFooterMenuDataSource.m

```diff
@@ -2,7 +2,7 @@
 #import "WMFArticleFooterMenuItem.h"
 #import "MWKArticle.h"
 #import "NSDate+Utilities.h"
-#import "WMFArticleFooterMenuCell.h"
+#import "WMFArticleListTableViewCell.h"
```

Comment:
It's basically at `WMF` "default cell" w/ an image & text view that's RTL compliant at this point.  I'd consider renaming rather than make a separate cell that does the same thing.